openid4vci/issuer-initiated/run-test.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ../../util.sh
+
+echo "------------------------------------"
+echo "Cleaning up running Docker containers and volumes, and key material..."
+echo "------------------------------------"
+docker compose down
+docker compose rm -f -v
+rm -rf ./node-*/data
+mkdir ./node-A/data ./node-B/data  # 'data' dirs will be created with root owner by docker if they do not exit. This creates permission issues on CI.
+
+echo "------------------------------------"
+echo "Starting Docker containers..."
+echo "------------------------------------"
+docker compose up -d
+
+waitForDCService nodeA
+waitForDCService nodeB
+
+echo "------------------------------------"
+echo "Creating NodeDIDs..."
+echo "------------------------------------"
+
+didNodeA=$(setupNode "http://localhost:11323" "nodeA:5555")
+printf "NodeDID for node-a: %s\n" "$didNodeA"
+
+# Wait for node B to receive the TXs created by node A, indicating the connection is working
+waitForTXCount "NodeB" "http://localhost:21323/status/diagnostics" 2 10
+
+didNodeB=$(setupNode "http://localhost:21323" "nodeB:5555")
+printf "NodeDID for node-b: %s\n" "$didNodeB"
+registerStringService "http://localhost:21323" "$didNodeB" "oidc4vci-wallet-metadata" "http://nodeB:1323/identity/$didNodeB/.well-known/openid-credential-wallet"
+
+# Wait for node A to receive all TXs created by node B
+waitForTXCount "NodeA" "http://localhost:11323/status/diagnostics" 5 10
+
+echo "------------------------------------"
+echo "Issuing credential..."
+echo "------------------------------------"
+vcNodeA=$(createAuthCredential "http://localhost:11323" "$didNodeA" "$didNodeB")
+printf "VC issued by node A: %s\n" "$vcNodeA"
+
+waitForDiagnostic "nodeA" issued_credentials_count 1
+waitForDiagnostic "nodeB" credential_count 1
```

Comment:
check the number of transaction to make sure it was shared on the DAG?

Reply:
It should not be shared on the DAG, since it must be issued using OpenID4VCI. This is asserted by not configuring node DID, which prevents private transactions from being exchanged.